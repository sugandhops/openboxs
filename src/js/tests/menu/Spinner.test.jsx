import React from 'react';

import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';

import Spinner from 'components/spinner/Spinner';

describe('spinner component', () => {
  it('should match snapshot', () => {
    const renderedSpinner = renderer.create(<Spinner />)
      .toJSON();
    expect(renderedSpinner)
      .toMatchSnapshot();
  });

  it('should render component correctly', () => {
    render(<Spinner />);
    const spinnerText = screen.getByText('Loading...');
    expect(spinnerText)
      .toBeTruthy();
  });
});
