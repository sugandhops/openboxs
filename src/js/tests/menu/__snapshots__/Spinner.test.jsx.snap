// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`spinner component should match snapshot 1`] = `
<div
  className="d-flex w-100 h-100 justify-content-center align-items-center"
>
  <div
    className="circle-spinner spinner-border"
    role="status"
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
